File: 3-Bimestre/Turma-A/ex-4.py
contador = 1
soma = 0

n = int(input('Digite um número positivo: '))
if n > 0:
    while contador <= n:
        if contador % 2 == 0:
            soma += contador
        contador += 1
    print(f'A soma dos números pares é: {soma}')
else:
    print('Número inválido')

File: 3-Bimestre/ex-4.cpp
#include <iostream>
using namespace std;

int main() {
    int n, contador = 1, soma = 0;
    cout << "Digite um numero positivo: ";
    cin >> n;

    if (n > 0) {
        while (contador <= n) {
            if (contador % 2 == 0)
                soma = soma + contador;
            contador++;
        }
        cout << "Soma dos numeros pares: " << soma;
    } else {
        cout << "Numero invalido";
    }

    return 0;
}
